src/Bicep.LangServer.IntegrationTests/CompletionTests.cs

```diff
@@ -304,6 +304,50 @@ public async Task Completions_are_not_offered_immediately_after_open_paren()
             await RunCompletionScenarioTest(this.TestContext, ServerWithBuiltInTypes, fileWithCursors, AssertAllCompletionsEmpty, '|');
         }
 
+    [DataRow(
```

Comment:
Personally I feel like `DataRow` is best avoided unless you're sure it's necessary to pass in different sets of params into the test. It makes it harder to understand which value corresponds to which variable.

If the goal is to save on line count, there are other ways, such as creating a reusable method (e.g. `RunCompletionScenarioTest`)